x/group/spec/04_events.md

```diff
@@ -8,56 +8,64 @@ The group module emits the following events:
 
-| Type                                     | Attribute Key | Attribute Value                          |
-|------------------------------------------|---------------|------------------------------------------|
-| message                                  | action        | /cosmos.group.v1beta1.Msg/CreateProposal |
-| cosmos.group.v1beta1.EventCreateProposal | proposal_id   | {proposalId}                             |
+| Type                                | Attribute Key | Attribute Value                     |
+| ----------------------------------- | ------------- | ----------------------------------- |
+| message                             | action        | /cosmos.group.v1.Msg/CreateProposal |
+| cosmos.group.v1.EventCreateProposal | proposal_id   | {proposalId}                        |
 
 ## EventWithdrawProposal
 
-| Type                                       | Attribute Key | Attribute Value                            |
-|--------------------------------------------|---------------|--------------------------------------------|
-| message                                    | action        | /cosmos.group.v1beta1.Msg/WithdrawProposal |
-| cosmos.group.v1beta1.EventWithdrawProposal | proposal_id   | {proposalId}                               |
+| Type                                  | Attribute Key | Attribute Value                       |
+| ------------------------------------- | ------------- | ------------------------------------- |
+| message                               | action        | /cosmos.group.v1.Msg/WithdrawProposal |
+| cosmos.group.v1.EventWithdrawProposal | proposal_id   | {proposalId}                          |
 
 ## EventVote
 
-| Type                           | Attribute Key | Attribute Value                |
-|--------------------------------|---------------|--------------------------------|
-| message                        | action        | /cosmos.group.v1beta1.Msg/Vote |
-| cosmos.group.v1beta1.EventVote | proposal_id   | {proposalId}                   |
+| Type                      | Attribute Key | Attribute Value           |
+| ------------------------- | ------------- | ------------------------- |
+| message                   | action        | /cosmos.group.v1.Msg/Vote |
+| cosmos.group.v1.EventVote | proposal_id   | {proposalId}              |
 
 ## EventExec
 
-| Type                           | Attribute Key | Attribute Value                |
-|--------------------------------|---------------|--------------------------------|
-| message                        | action        | /cosmos.group.v1beta1.Msg/Exec |
-| cosmos.group.v1beta1.EventExec | proposal_id   | {proposalId}                   |
+| Type                      | Attribute Key | Attribute Value           |
+| ------------------------- | ------------- | ------------------------- |
+| message                   | action        | /cosmos.group.v1.Msg/Exec |
+| cosmos.group.v1.EventExec | proposal_id   | {proposalId}              |
```

Comment:
should we also add result and logs in there since there have been added in https://github.com/cosmos/cosmos-sdk/pull/11559?